core/src/main/java/dev/keva/core/command/impl/string/GetEx.java

```diff
@@ -0,0 +1,109 @@
+package dev.keva.core.command.impl.string;
+
+import dev.keva.core.command.annotation.CommandImpl;
+import dev.keva.core.command.annotation.Execute;
+import dev.keva.core.command.annotation.ParamLength;
+import dev.keva.core.exception.CommandException;
+import dev.keva.ioc.annotation.Autowired;
+import dev.keva.ioc.annotation.Component;
+import dev.keva.protocol.resp.reply.BulkReply;
+import dev.keva.protocol.resp.reply.Reply;
+import dev.keva.store.KevaDatabase;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Timestamp;
+
+@Component
+@CommandImpl("getex")
+@ParamLength(type = ParamLength.Type.AT_LEAST, value = 2)
+public class GetEx {
+    private final KevaDatabase database;
+
+    private final String EX = "EX";
+    private final String PX = "PX";
+    private final String EXAT = "EXAT";
+    private final String PXAT = "PXAT";
+    private final String PERSIST = "PERSIST";
+
+    private int EX_FLG = 1 << 2;
+    private int PX_FLG = 1 << 3;
+    private int EXAT_FLG = 1 << 6;
+    private int PXAT_FLG = 1 << 7;
+    private int PERSIST_FLG = 1 << 8;
+
+    @Autowired
+    public GetEx(KevaDatabase database) {
+        this.database = database;
+    }
+
+    @Execute
+    public Reply<?> execute(byte[][] params) {
```

Comment:
This should be `BulkReply` right, not `Reply<?>` because in this method we always return BulkReply @yampml